README.md

```diff
@@ -5,44 +5,43 @@ Our open source community's Wiki focuses on answering your questions about learn
 - Curriculum
 - Other Programming Languages
 - Local Campsite Communities
-- Nonprofit Projects  
- ...and much more
+- Nonprofit Projects and much more
 
-*Taste* the wiki at [**FreeCodeCamp/wiki**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki) :yum: 
+*Taste* the wiki at [**FreeCodeCamp/wiki**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki) :yum:
 
-## 
+## Steps to follow
 
-### 1. **Fork it** :fork_and_knife:   
+### 1. **Fork it** :fork_and_knife:
 You can get your own fork/copy of [FreeCodeCamp wiki](https://github.com/FreeCodeCamp/wiki) by using the <a href="https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box"><kbd><b>Fork</b></kbd></a> button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
 
 > [![Fork Button](https://help.github.com/assets/images/help/repository/fork_button.jpg)](https://github.com/FreeCodeCamp/wiki)
 
-### 2. **Clone it** :busts_in_silhouette:  
+### 2. **Clone it** :busts_in_silhouette:
 Once you have forked the wiki repository in Github, you need to clone/download it to local machine with...
-  
+
 ```sh
 $ git clone https://github.com/YOUR_USERNAME/wiki.git
 ```
 
 > *this makes a local copy in your machine.*
 
 ### 3. **Set it up** :up:
-Run the following commands to see that _your local copy_ has a reference to _your forked remote repository_ in Github :octocat:  
+Run the following commands to see that _your local copy_ has a reference to _your forked remote repository_ in Github :octocat:
 
 ```sh
 $ git remote -v
 origin  https://github.com/YOUR_USERNAME/wiki.git (fetch)
 origin  https://github.com/YOUR_USERNAME/wiki.git (push)
 ```
 
-Magic :sparkles: :sparkles: :eyes:  
+Magic :sparkles: :sparkles: :eyes:
```

Comment:
@Rafase282 apart from the commented ones, other spaces were not required. These spaces which i've commented about are required.